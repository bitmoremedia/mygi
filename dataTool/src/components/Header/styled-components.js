import styled from 'styled-components'

export const Container = styled.div`
  padding: 10px;
  background-color: red;
  height: 60px;
`

export const Heading = styled.h1`
  margin-top: 12px;
`
