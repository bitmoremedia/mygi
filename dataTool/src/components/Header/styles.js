import styled from 'styled-components';

export const Container = styled.div`
  padding: 5px;
  background-color: red;
  height: 80px;
`;
