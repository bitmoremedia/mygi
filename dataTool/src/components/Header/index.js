import React from 'react'

import { Row, Col } from '../common/Grid'
import { Container, Heading } from './styled-components'

const Header = () =>
  <Container>
    <Row>
      <Col xs={12}>
        <Heading>MyGi Data Tool</Heading>
      </Col>
    </Row>
  </Container>

export default Header
