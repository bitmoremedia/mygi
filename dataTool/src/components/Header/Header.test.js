import React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import Header from './index'

test('component renders without error', () => {
  const wrapper = mount(
    <Header />,
  )
  expect(wrapper.exists()).toBe(true)
})

// SNAPSHOT TESTS

test('component renders as expected', () => {
  const wrapper = mount(
    <Header />,
  )
  expect(toJson(wrapper)).toMatchSnapshot()
})
