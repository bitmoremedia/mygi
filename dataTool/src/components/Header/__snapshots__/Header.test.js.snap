exports[`test component renders as expected 1`] = `
<Component>
  <styled.div>
    <div
      className="jTlqhU">
      <Row>
        <Component
          className="jhQCSn">
          <div
            className="jhQCSn">
            <Col
              xs={12}>
              <Component
                className="egpniL"
                xs={12}>
                <div
                  className="egpniL">
                  <styled.h1>
                    <h1
                      className="iZkySE">
                      MyGi Data Tool
                    </h1>
                  </styled.h1>
                </div>
              </Component>
            </Col>
          </div>
        </Component>
      </Row>
    </div>
  </styled.div>
</Component>
`;
